command_line/find_spots.py

```diff
@@ -69,6 +69,11 @@
       .help = "The log filename"
   }
 
+  maximum_trusted_value = 0
+    .type = float
+    .help = "Override maximum trusted value for spot finding only"
+    .expert_level = 2
+
```

Comment:
Good suggestion, implemented

Reply:
As far as I understand the `.optional` attribute is irrelevant here and has no effect - it only affects `choice` types and when scopes can be multiple (and when it _is_ relevant, the default is `.optional = True` anyway). I would suggest it is better not to add `.optional` here, as it might confuse future developers into thinking that it is required or having it somehow affects the behaviour.

Reply:
Ah, well I count myself among the confused then.